### PR TITLE
fix(Log class): Log postponed to wp_footer action

### PR DESCRIPTION
**Pull request by vandebled**
Log postponed to wp_footer action, but can also be bypassed

### CODE REVIEW

File: lib/Utils/Log.php [hunk 1]
```diff
@@ -4,36 +4,56 @@
...
+    }
+
+    public static function consoleDebug($data, $postpone, $title = 'PHP', $logType = 'log')
     {
         $title .= '(' . self::getCallerFile(2) .'):';
         $type = gettype($data);
         if (is_array($data) || is_object($data)) {
             $output = json_encode($data);
-            echo "<script>console.$logType('$title', '($type)', $output);</script>\n";
+            $result =  "<script>console.$logType('$title', '($type)', $output);</script>\n";
```
Comment (domtra) [inline lib/Utils/Log.php]: please you `"{$var}"` syntax for consistency

File: lib/Utils/Log.php [hunk 2]
```diff
@@ -4,36 +4,56 @@
...
+        self::echoDebug($result, $postpone);
     }
 
-    public static function pp($data)
+    public static function printDebug($data, $postpone)
     {
         $type = gettype($data);
-        echo "<pre>";
-        echo "(" . $type . ") ";
+        $output = "<pre>";
```
Comment (domtra) [inline lib/Utils/Log.php]: always use single quotes unless double quotes are necessary.

File: lib/Utils/Log.php [hunk 3]
```diff
@@ -4,36 +4,56 @@
 
 class Log
 {
-    public static function console($data)
+    public static function console($data, $postpone = true)
     {
-        self::consoleDebug($data);
+        $data = self::consoleDebug($data, $postpone);
```
Comment (domtra) [inline lib/Utils/Log.php]: the assignment is not necessary

**Comment by vandebled** [comment]
@domtra all changes done

**Pull request merged**